api/container/containerv2/workers.go

```diff
@@ -133,7 +135,18 @@ func (r *worker) ListWorkers(clusterIDOrName string, showDeleted bool, target Cl
 	return workers, err
 }
 
-//Get ...
+// ListClassicWorkers ...
+func (r *worker) ListClassicWorkers(clusterIDOrName string, showDeleted bool, target ClusterTargetHeader) ([]Worker, error) {
```

Comment:
can you add a UT for this please?

Reply:
Added in https://github.com/IBM-Cloud/bluemix-go/pull/448/commits/ea640b4e43b57e8d8aec8156aa22c08c1785b1bb

```
% go test ./api/container/containerv2/ -ginkgo.focus "ListClassicWorkers"   
ok      github.com/IBM-Cloud/bluemix-go/api/container/containerv2       90.500s
```